sample.html

```diff
@@ -143,6 +145,13 @@ <h1 class="form-name">Form Name</h1>
           throw new Error("Please enter a valid email")
         }
       })
+
+      validate("github", required("Github User Profile Link is required"), async value => {
+        const githubformat = /^[h][t][t][p](s?)[:][/][/]github\.com[/]|^github\.com[/]/
```

Comment:
you dont need to have brackets around a single character. Also, it only matches the github link, which means i can just pass in the github.com/ and it will show no error.
![image](https://user-images.githubusercontent.com/41939011/189527424-6cf4afa5-4860-4f75-8ee2-006e0994020c.png)
![image](https://user-images.githubusercontent.com/41939011/189527437-884002e3-214f-4fd4-8dd9-05e4ab6332cb.png)